handbook/people-ops/spending-company-money.md

```diff
@@ -1,3 +1,67 @@
 # Spending company money
+Please see below for a list of reimbursable expenses (all in USD).
+
+## One-time expenses
+**Laptop (engineers):** $3500
+
+**Laptop (non-engineers):** $2000
+
+**Laptop coverage:** $380
+
+**Monitor:** $380
+
+**Headphones:** $200
+
+**Keyboard:** $130
+
+**Mouse or trackpad:** $80 or $130
+
+**USB-C Adapter:** $70
+
+**HDMI/monitor cable:** $15
+
+**Laptop stand:** $40
+
+**Desk:** $500
+
+**Ergonomic chair:** $200
+
+
+
+## Ongoing expenses
+**Software:** case by case, please get your manager’s approval.
+
+**Internet/WiFi plan:** $60/month
```

Comment:
Should we make people expense this, or just add it as a default like the cell phone plan? I vote the latter to avoid having to find the receipt every month. And let's reduce to $50, given that.

Reply:
My preference would be the latter FWIW.

Reply:
Agree.

Reply:
To be noted that adding to salary means it becomes taxable income, rather than expensable.

Reply:
Expensing is a pain and time consuming. If we intend to compensate people for internet because it is necessary for the job, then it doesn't make sense to have the whole company waste time every month uploading a receipt to Expensify. I think this is less about tax efficiency (if that is what you are worried about) and more about communicating "your job requires broadband internet access and we compensate for that"

Reply:
@tsenart's point is still valid though, isn't it? Back of the envelope calculation: In Germany you get the highest possible income tax of 42% if you earn anything over €54,950/year. $50 are €45 according to Google (not considering Veem's markup). So real money available to pay the broadband bill after taxation is €26. A basic 50 MBit/s at Deutsche Telekom cost €40/month, or 45€ for 100MBit/s for those all-company Zoom meetings. So then it wouldn't really fulfill the intend, unless it's higher.